app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<p>Here is a list of users for your Blog app</p>
\ No newline at end of file
+<a href="<%= user_posts_path(@user) %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<div class="custom-user-box">
+  <%= render partial: 'user_helper/user_photo', locals: { user: @user } %>
+  <%= render partial: 'user_helper/user_details', locals: { user: @user } %>
+</div>
+<div class="custom-bio">
+  <h3>Bio</h3>
+  <p>
+    <%= @user.bio %>
+  </p>
+</div>
+<% @user.recent_posts.each do |post| %>
+  <div class="custom-post custom-bio">
+    <h3>Post <%= raw "#" + post.id.to_s %></h3>
+    <p><%= post.text %></p>
+    <div class='custom-post-interactions'>
+      <span>Comments: <%= post.comments_counter %>,</span>
+      <span>Likes: <%= post.likes_counter %></span>
```

Comment:
- [ ] Also you forgot to change your spelling here, I missed it the first time because you were not rendering your user-show page, kindly change `post.comments_counter` to `post.commentsCounter` and  `post.likes_counter `to `post.likesCounter` These are causing errors, kindly fix them
![Screenshot from 2023-10-05 23-43-46](https://github.com/leloufadel/my_blog_app/assets/66097371/78310b1c-2fdf-47e9-9249-29f9074a2f1b)

Reply:
@VICTOR-CHUKWU Thank you for your comments, could you please check if it is working, so I can ask for review?